acceptance/setup/aio/pre-suite/015_PackageHostsPresets.rb

```diff
@@ -1 +1,9 @@
-master['use-service'] = true
+if options['is_puppetserver']
+  master['use-service'] = true
+  master['puppetservice'] = 'puppetserver'
+  master['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
I see what you are doing now. That's a bit confusing (two interface entry points).  But, i don't really have a better solution, other than replicating it all over to another options/setup dir <sigh>.